doc/classes/Object.xml

```diff
@@ -1041,7 +1043,7 @@
 			<param index="0" name="property_path" type="NodePath" />
 			<param index="1" name="value" type="Variant" />
 			<description>
-				Assigns a new [param value] to the property identified by the [param property_path]. The path should be a [NodePath] relative to this object, and can use the colon character ([code]:[/code]) to access nested properties.
+				Assigns a new [param value] to the property identified by the [param property_path]. The path should be a [NodePath] relative to this object, and can use the colon character ([code]:[/code]) to access nested properties. If the property or any of the path components do not exist or the given [param value]'s type doesn't match, nothing happens.
```

Comment:
It's a technicality. The beginning of the description calls it "property". I am aware that properties are an Object-exclusive concept in a way, hence a vector's components are not properties in the Godot sense, but for simplicity sake it's better to stick to it, I feel.
```suggestion
				Assigns a new [param value] to the property identified by the [param property_path]. The path should be a [NodePath] relative to this object, and can use the colon character ([code]:[/code]) to access nested properties. If the property does not exist or the given [param value]'s type doesn't match, nothing happens.
```

For similar simplicity, NodePath's documentation also states that "subnames" (what you called "path components" here) point to properties.